exporter/datadogexporter/internal/metrics/consumer.go

```diff
@@ -59,9 +62,19 @@ func (c *Consumer) toDataType(dt translator.MetricDataType) (out MetricDataType)
 // runningMetrics gets the running metrics for the exporter.
 func (c *Consumer) runningMetrics(timestamp uint64, buildInfo component.BuildInfo) (series []datadog.Metric) {
 	for host := range c.seenHosts {
-		// Report the host as running
-		runningMetric := DefaultMetrics("metrics", host, timestamp, buildInfo)
-		series = append(series, runningMetric...)
+		if host != "" {
```

Comment:
Is that needed? We already don't add empty hostnames to `seenHosts` due to the changes in `metrics_translator.go` (we don't call `ConsumeHost` on empty hostnames), so it feels like we're doing the same check twice.

Reply:
Not needed, I removed it